src/test/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervalsIntegrationTest.java

```diff
@@ -0,0 +1,146 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import htsjdk.samtools.util.IntervalList;
+import htsjdk.samtools.util.Interval;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.io.File;
+
+public final class PreprocessIntervalsIntegrationTest extends CommandLineProgramTest {
+    private static final File REFERENCE = new File(b37_reference_20_21);
+
+    @DataProvider(name = "intervalInputsFromCommandLine")
+    public final Object[][] testData() {
+        // Test for separate intervals
+        final int binLengthSeparateIntervalTest = 10_000;
+        final int paddingSeparateIntervalTest = 0;
+        final String[] outputFileNameSeparateIntervalTest =  {"GATK-preprocess-intervals-test", ".tmp"};
+        final ArrayList<Interval> inputIntervalsSeparateIntervalTest = new ArrayList<Interval>();
```

Comment:
you can just write  `new ArrayList<>();` 
Java automatically infers it

Reply:
As we discussed, you can just use `final List<Interval> intervals = Arrays.asList(...);`.

Reply:
Thanks @asmirnov239 for your suggestion. I went with @samuelklee's suggestion based on my private discussion with him.